### PR TITLE
Much more then null checking :)

### PR DESCRIPTION
**Pull request by shlomieYaniv**
- reformatted the entire solution
- added .gitognore
- added tests
- refactor some code for Java 11

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,71 +1,119 @@
...
+                    <version>${maven.surefire.version}</version>
+                    <configuration>
+                        <parallel>classes</parallel>
+                        <threadCount>6</threadCount>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <!--            <plugin>-->
+        <!--                <artifactId>spotless-maven-plugin</artifactId>-->
```
Comment (liorc95) [inline pom.xml:53]: Please return spotless

File: src/main/java/com/wss/remediation/Utils.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.wss.remediation;
+
+import org.owasp.esapi.ESAPI;
+import org.owasp.esapi.codecs.AbstractCharacterCodec;
+import org.owasp.esapi.codecs.UnixCodec;
+
+public class Utils {
+
+
+    static String EsapiEncoder(AbstractCharacterCodec codec, String param) {
```
Comment (liorc95) [inline src/main/java/com/wss/remediation/Utils.java:10]: please start a method name with a small letter

File: src/test/java/com/wss/remediation/WhiteSourceSanitizerTest.java [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+    @Test
+    void OSParameterEncoder_null_successfully() {
+        // Arrange
+
+        // Act
+        Assertions.assertThrows(NullPointerException.class,
+                () -> WhiteSourceSanitizer.OSParameterEncoder(null));
+
+        // Assert
```
Comment (liorc95) [inline src/test/java/com/wss/remediation/WhiteSourceSanitizerTest.java:58]: remove comment here

**Review by liorc95 — COMMENTED** [review]
Nice :)

File: src/test/java/com/wss/remediation/WhiteSourceSanitizerTest.java (hunk 3 above)
Comment (shlomieYaniv) [reply src/test/java/com/wss/remediation/WhiteSourceSanitizerTest.java:58]: comments removed

File: src/main/java/com/wss/remediation/Utils.java (hunk 2 above)
Comment (shlomieYaniv) [reply src/main/java/com/wss/remediation/Utils.java]: Change to lower letter

File: pom.xml (hunk 1 above)
Comment (shlomieYaniv) [reply pom.xml]: Spotless has returned :)
Didn't worked earlier don't know why.

**Pull request merged**